app/views/admin/workshops/_edit_form.html.haml

```diff
@@ -11,10 +11,19 @@
   .row
     .large-6.columns
       = f.input :host, as: :select, collection: Sponsor.all, include_blank: true, selected: (@workshop.host.id rescue '')
+  %br
   .row
     .large-6.columns
       = f.association :sponsors
   %br
+  .row
+    .large-6.columns
+      %h6 Default description text:
+      %p= t("workshops.lead")
+    .large-6.columns
+      = f.input :description, as: :text, label: 'Additional description',
+      placeholder: 'Use this space to write a description of the workshop. You can use HTML here!'
```

Comment:
@turger can we move the bit about HTML support out from the placeholder and into the label? For example `Additional description (supports HTML)`.

Reply:
Good idea, I'll do the change.